src/cljs/rems/fields.cljs

```diff
@@ -142,24 +142,30 @@
   (.. event -target -value))
 
 (defn text-field
-  [{:keys [validation on-change] :as opts}]
+  [{:keys [validation on-change collapse-info info-text] :as opts}]
```

Comment:
There should be no need to have both `collapse-info` and `info-text`. Collapse can be hidden if there is no `info-text` given?